core/server/src/main/java/alluxio/master/AlluxioMaster.java

```diff
@@ -442,6 +435,10 @@ protected void startServing(String startMessage, String stopMessage) {
   }
 
   protected void startServingWebServer() {
+    mWebServer = new MasterUIWebServer(ServiceType.MASTER_WEB.getServiceName(),
```

Comment:
Right. I'm going for the path of least resistance here on getting master fault tolerance fixed. I think we definitely need a followup PR which makes sense of all the different lifecycle.